svthalia/concrexit
# Translate collaboration email address

Closes #1080 

### Summary
This translates the contact information for companies.

### How to test
Steps to test the changes you made:
1. Go to the partner pages and check the English and Dutch email address
2. Go to the contact page and check the English and Dutch email address
